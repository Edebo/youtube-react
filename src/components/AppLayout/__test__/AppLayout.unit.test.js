import React from "react";

import { shallow } from "enzyme";
import { AppLayout } from "../AppLayout";

describe("testing app layout snapshot", () => {
  test("rendering empty app layout", () => {
    const wrapper = shallow(<AppLayout />);

    expect(wrapper).toMatchSnapshot();
  });

  test("rendering app layout with a props children", () => {
    const wrapper = shallow(
      <AppLayout>
        <div>Hello world</div>
      </AppLayout>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
