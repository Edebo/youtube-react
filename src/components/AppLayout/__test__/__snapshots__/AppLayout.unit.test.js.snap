// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing app layout snapshot rendering app layout with a props children 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Hello world
  </div>
</div>
`;

exports[`testing app layout snapshot rendering empty app layout 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
</div>
`;
